### PR TITLE
Refactored UnicodeToLatexFormatterTest 

### PR DESCRIPTION
**Pull request by ningxie1991**
This pull request contributes to issue #6207, which is to add more unit tests or improve existing ones. I used Pitest to compute the line and mutation coverage. The empty string input was missing in the test, as well as the boundary case test for high code point unicode inputs. 

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/test/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatterTest.java [hunk 1]
```diff
@@ -14,23 +18,21 @@ void setUp() {
...
-    void formatMultipleUnicodeCharacters() {
-        assertEquals("{{\\aa}}{\\\"{a}}{\\\"{o}}", formatter.format("\u00E5\u00E4\u00F6"));
+    private static Stream<Arguments> testCases() {
+        return Stream.of(
+                Arguments.of("", ""), // empty string input
+                Arguments.of("abc", "abc"), // non unicode input
+                Arguments.of("\u00E5\u00E4\u00F6", "{{\\aa}}{\\\"{a}}{\\\"{o}}"), // multiple unicodes input
+                Arguments.of("\u0081", ""), // high code point unicode, boundary case: cp = 129
+                Arguments.of("\u0080", ""), // high code point unicode, boundary case: cp = 128 < 129
+                Arguments.of(new UnicodeToLatexFormatter().getExampleInput(), "M{\\\"{o}}nch")
```
Comment (koppor) [inline src/test/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatterTest.java:28]: Please switch parameters to comply with `assertEquals` semantics: first the `expected`, then the `input` value.

**Review by koppor — CHANGES_REQUESTED** [review]

File: src/test/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatterTest.java (hunk 1 above)
Comment (ningxie1991) [reply src/test/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatterTest.java]: Code is updated as suggested. :)

**Review by Siedlerchr — APPROVED** [review]
See checkstyle/reviewdog. Otherwise is good

**Comment by koppor** [comment]
@ningxie1991 Maybe, we need to guide you how to read the checkers of GitHub. I marked the required tests:

grafik

They are all required. If they are green, they are good to go. If they are yellow, one has to wait (and can do nothing currently). If they are red, one has to work on them. Thus, please work on checkstyle. In case you followed our guidelines to setup the local workspace, <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>L</kbd> should reformat the code properly.

**Comment by koppor** [comment]
We closed the PR, because there was no activity and the changes were rather small.

In case you are still interested in contributing, please try to work on another issue.

**Comment by Siedlerchr** [comment]
I fixed the checkstlye issue and merged it https://github.com/JabRef/jabref/commit/0c8662ea1df49bb4598d5766bda1cf5821f0e9c8

**Comment by ningxie1991** [comment]
> We closed the PR, because there was no activity and the changes were rather small.
> 
> In case you are still interested in contributing, please try to work on another issue.

Really sorry that I lost track of this pull request. I will make sure required tests pass in future pull request.

**Comment by ningxie1991** [comment]
> I fixed the checkstlye issue and merged it [0c8662e](https://github.com/JabRef/jabref/commit/0c8662ea1df49bb4598d5766bda1cf5821f0e9c8)

Thank you so much!

**Pull request closed without merging**